src/wallet/wallet.cpp

```diff
@@ -943,6 +944,20 @@ bool CWallet::LoadUnifiedCaches()
         }
     }
 
+    // Restore decrypted Orchard notes.
```

Comment:
I wonder if we should rename `LoadUnifiedCaches` to `LoadCaches` since now it's a catchall for all restoration logic.